src/components/LoginMapButton.js

```diff
@@ -0,0 +1,10 @@
+import React, { PureComponent } from 'react';
+import './LoginMapButton.css';
+
+export default class LoginMapButton extends PureComponent {
+  render() {
+    return (
```

Comment:
In Overview.js you are handing the LoginMapButton a prop `onClick` here you are not doing anything with this prop so it will do nothing. 

I think your intent was that when the user clicks the onClick is called. 
Therefor grab the variable that is passed to the component via its props
`const { onClick } = this.props;`